app/src/main/java/tech/ula/utils/DownloadUtility.kt

```diff
@@ -7,36 +7,43 @@ import android.net.NetworkInfo
 import android.net.Uri
 import android.os.Environment
 import java.io.File
+import java.net.HttpURLConnection
+import java.net.URL
 
 class DownloadUtility(val context: Context, val archType: String, distType: String) {
 
+    companion object {
+        val branch = "master"
+    }
```

Comment:
I think this can just be a private val. I'm not seeing it used in any other files.

Reply:
true